docs/docs/using-client-side-only-libraries.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using client side only packages
+issue: https://github.com/gatsbyjs/gatsby/issues/13355
+---
+
+This is a stub. Help our community expand it.
+
+Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
+pull request gets accepted.
+
+On occasion, you may need to use a function or library that only works client side. This usually is because the library in question accesses something that isn't available during server-side rendering (SSR).
+
+You'll need to use one of the workarounds outlined below if your project fails to compile with `gatsby develop` or `gatsby build` with an error like:
+
+```bash
+Reference error: Window is not Defined
+```
+
+## Workaround 1: Use a different library or approach
+
+Sometimes the simplest approach is to work around the problem. If you can re-implement your component using a plugin which _doesn't_ break SSR, that's probably best.
+
+## Workaround 2: Add client-side package via CDN
+
+In the component where you need it, load the package via CDN using a [`<script />`](https://www.w3schools.com/Tags/tag_script.asp) tag with `react-helmet`. You'll still need to check to see if `window` exists before using the library, so that SSR can still complete.
+
+```jsx
+import { Helmet } from 'react-helmet'
+
+
+const MyComponent = (props) => {
+  if (typeof window !== undefined) {
+    // check for external package and use as needed here
```

Comment:
What would be involved here?

Reply:
It depends on the specific use case - any function call on a dependency which talks to `window` would fail.  In the case of shopify, the imported library is made available as something like `window.shopifyUi`, and for Intercom, `window.Intercom`, for example.  Some dependencies call on window under the covers somewhere, like the example in the original post from [this issue](https://github.com/gatsbyjs/gatsby/issues/13355#issue-432952792).  

I wasn't really sure how to flesh out the example because of that - any particular thoughts on what might be best?

Reply:
It might be more clear when split into two comments:
```
// check for browser DOM readiness
if (typeof window !== undefined) {
    // use external package as needed here, e.g. `window.externalLibrary.method()`
```
What do you think?